src/script.js

```diff
@@ -0,0 +1,39 @@
+const bscript = require('bscript-parser')
+
+class Script {
+  get length () {
```

Comment:
As a matter of style, I prefer the constructor to be the first function definition in a class. Not required though.

Reply:
that's fine i'll move it up